app/views/schools/participants/index.html.erb

```diff
@@ -1,14 +1,13 @@
 <% content_for :title, "Early career teachers and mentors" %>
 
-<% content_for :before_content, govuk_back_link(
-  text: "Back",
-  href: schools_cohort_path(@cohort.display_name))
-%>
+<% content_for :before_content, govuk_breadcrumbs(
+  breadcrumbs: { @school.name => schools_dashboard_path, "#{@cohort.display_name} cohort" => schools_cohort_path }
```

Comment:
The breadcrumbs seem to repeat a lot. On E&L we made a helper to generate them for CIP materials. Maybe not worth doing here, but something to think about when we touch them next.

Reply:
Will take a look in the next one, I'm doing some breadcrumb logic there anyway